src/sec_edgar_bulker/core/models.py

```
@@ -0,0 +1,499 @@
+    """Custom section for header parsing.
+    
+    Attributes:
+        name: Section name.
+        start: Start pattern.
+        end: End pattern.
+        fields: List of field patterns.
+    """
+    name: str
+    start: str
+    end: str
+    fields: List[Dict[str, str]]
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)